Textareas uploading and downloading. Unicode supporting.

username1565 opened the pull request:
Now, all textareas can be filled by loading files.
This is TXT, .ASC with keys, or another files, containing the text.
After loading file again - textarea text updating.

Results can be download as TXT files, when textareas not empty.
Buttons for downloading available after textareas containing any data.
Automatically updating links after changing text in textareas.
Supporting .asc files as key containers and another files containing text.
Supporting UTF-8 and UNICODE encoding and whitespace symbols.

test: ('늞⍪䪄㣣錋뢲⚬깾尙ຝ䑺噑惊ሆ漅₽娛䐀䊣ⲝ砧瓭簤Ḳᗈ鐏象'===
'늞⍪䪄㣣錋뢲⚬깾尙ຝ䑺噑惊ሆ漅₽娛䐀䊣ⲝ砧瓭簤Ḳᗈ鐏象')=== true

username1565 committed:
Textareas uploading and downloading

	- openFile function moved here from index.htm. Function need to load textareas from files. .InnerText changed to .value
	- added here the functions to update download-buttons when textareas not empty.
	- runing this functions inside this script, when data changing in textareas.

username1565 committed:
Replace 4 buttons

Delete openFile function - this is replaced to main.js now.

Delete doDL function - this do downloading file, without filename.
This buttons was been available always, and sometimes downloaded empty file.

Add commented "Guide" tab, because this is exists, but empty. This tab not found in the code.

Add buttons after textareas for downloading it as text.
Now, when readonly textareas have text - there is possible to download it by click button.

+ previous fixes included.

TheChiefMeat commented:
I can't seem to get the "Receiver's Private Key (decryption)" to paste the files content into the window. If you could fix that then I can merge this :)

TheChiefMeat commented:
Also all future edits will be done to the included index.html instead of the index.htm file for better compatibility, I'll delete the index.htm file once this commit is done.

username1565 committed:
fix textarea filling

fixed

username1565 committed:
Delete index.htm

This will be index.html

username1565 committed:
Create index.html

According the comments in this pull request: https://github.com/TheChiefMeat/pgp/pull/3

username1565 commented:
Done.

username1565 committed:
Update index.html

Fix text.
